tests/sentry/api/endpoints/test_user_details.py

```diff
@@ -133,8 +111,7 @@ def test_change_username_when_different(self):
         user = self.create_user(email="c@example.com", username="diff@example.com")
         self.login_as(user=user, superuser=False)
 
-        resp = self.client.put(self.url, data={"username": "new@example.com"})
-        assert resp.status_code == 200, resp.content
+        self.get_valid_response("me", username="new@example.com")
```

Comment:
what is "me" being passed in for?

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/user.py#L56
When the base user endpoint gets arguments from the URL, it knows to replace `"me"` with `request.user.id`. This isn't explained anywhere so I'm going to add a line of documentation.